src/ucp/tag/tag_send.c

```diff
@@ -307,7 +339,10 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_send_sync_nb,
     ret = ucp_tag_send_req(req, count, &ucp_ep_config(ep)->tag.eager,
                            ucp_ep_config(ep)->tag.rndv.rma_thresh,
                            ucp_ep_config(ep)->tag.rndv.am_thresh,
-                           cb, ucp_ep_config(ep)->tag.sync_proto, 1);
+                           param->cb.send, param->user_data,
+                           ucp_ep_config(ep)->tag.sync_proto,
+                           !(param->attr_mask & UCP_OP_ATTR_FIELD_REQUEST),
```

Comment:
please comment `TODO: fix here`, because it mimics the current `send_nbr` behavior, but it would be more correct to introduce new op flag, like `UCP_OP_ATTR_FLAG_NO_ZCOPY`, which would not use zcopy protocol

Reply:
added new flag